charts/k8ssandra/templates/medusa/medusa-config.yaml

```diff
@@ -1,10 +1,10 @@
-{{- $bucketStorageTypes :=  list "s3" "gcs" -}}
+{{- $bucketStorageTypes :=  list "s3" "google_storage" -}}
```

Comment:
I think this is wrong now? Or is there a reason to have these two only listed here when we also support ``s3_compatible`` and ``local`` ?

Reply:
Although I'm not sure if this is used at all.. maybe this should be a map and the if clause should check for existence of key

Reply:
Good question. I wasn't sure. It is only used here now:

```
{{- if and .Values.medusa.multiTenant (has .Values.medusa.storage $bucketStorageTypes)}}
    prefix = {{ .Values.clusterName }}.{{ .Release.Namespace }}
{{- end }}
```
I assume we do multi-tenant with `s3_compatible` and `local`. I will update.

Reply:
See line 39 for where it's used; it's already checking for existence in the list. Not sure what we'd gain from changing to a map.

Reply:
Changes pushed.